### DISCUSSION TITLE
Some ideas

### DISCUSSION THREAD

**Discussion by KylinShinlyLin** [flyerhq/flutter_chat_ui#687] [Q&A] [answered]
I think in order to accommodate both computer and other devices, the bubble cards should cover the interface as much as possible. There should ideally be no restriction on the maximum width, or the maximum width can be set by the individual. 

It would also be a good option for the bubbles to natively support Markdown format. However, Flutter has always had issues with multi-line copying in Markdown format.

**Comment by demchenkoalex** [comment, marked as answer]
> I think in order to accommodate both computer and other devices, the bubble cards should cover the interface as much as possible. There should ideally be no restriction on the maximum width, or the maximum width can be set by the individual.

I think it is like this in `v2` by default

> It would also be a good option for the bubbles to natively support Markdown format. However, Flutter has always had issues with multi-line copying in Markdown format.

I use `flutter_markdown` in `v2`, I know it is not ideal, but it is best Flutter provides and writing my own markdown parser is an overkill. Hopefully Flutter improves :)
